### PR TITLE
Fix: Text overflow issue on sidebar in environment page

### PR DESCRIPTION
**Pull request by Afrin127329**
Fixes issue #453 

Now it looks like this:

image

### CODE REVIEW

**Review by Stefterv — CHANGES_REQUESTED** [review]
Screenshot 2024-07-12 at 15 15 28
Look good, we're just missing the wrapping on the title

File: src/components/Sidebar.module.css [hunk 1]
```diff
@@ -49,6 +49,9 @@
   flex-grow: 1;
   position: relative;
   overflow-y: auto;
+  & ul li h4 {
```
Comment (SableRaf) [inline src/components/Sidebar.module.css:52]: This doesn't seem to target the sidebar module's title

**Review by SableRaf — CHANGES_REQUESTED** [review]

File: src/components/Sidebar.module.css (hunk 1 above)
Comment (Afrin127329) [reply src/components/Sidebar.module.css:52]: Working on this.

Comment (Afrin127329) [reply src/components/Sidebar.module.css:52]: Looks good to me. it has been applied to the targeted element

image

**Review by SableRaf — APPROVED** [review]

**Comment by SableRaf** [comment]
The changes look good to me! Thanks for your contribution @Afrin127329 ✨ @Stefterv have a look and feel free to merge.

**Comment by Stefterv** [comment]
Screenshot 2024-07-16 at 10 27 41
Yep, looks great!

**Review by Stefterv — APPROVED** [review]

**Pull request merged**